app/src/main/res/values/strings.xml

```diff
@@ -54,7 +54,7 @@
     <!-- activity upload -->
     <string name="activity_upload_upload_activity_title">New Activity</string>
     <string name="activity_upload_edit_text_title_hint">Title</string>
-    <string name="activity_upload_edit_text_nparticipants_hint">Number of participants</string>
```

Comment:
I think we can remove the line "<string name="greeting_message">Hello, %1$s!</string>" from `strings.xml` since this is a residual from the very first version of our app and we currently don't use it anymore!

Reply:
yes thanks ! I will change that before merge

Reply:
Yep, thank you!